libs/auth/src/certs.ts

```diff
@@ -39,7 +39,7 @@ export const STANDARD_CERT_FIELDS = [
 export interface CustomCertFields {
   component: 'card' | 'admin' | 'central-scan' | 'mark' | 'scan';
   jurisdiction: string;
-  cardType?: 'sa' | 'em' | 'pw';
+  cardType?: 'system-administrator' | 'election-manager' | 'poll-worker';
```

Comment:
Following up on an earlier piece of feedback to just use the full user role names since we're not strapped for space in the same way anymore